container-entrypoint.sh

```diff
@@ -22,6 +27,13 @@ else
   echo "SSL certificate database already exists. Skipping initialization."
 fi
 
+# Sidecar: per-site-exporter - stream access.log into the exporter
+if [ "${1:-squid}" = "per-site-exporter" ]; then
+  shift || true
+  [ -f "${ACCESS_LOG}" ] || touch "${ACCESS_LOG}"
+  tail -n +1 -F "${ACCESS_LOG}" | exec /usr/local/bin/per-site-exporter "$@"
```

Comment:
I think there's a subtle point here: 

If I understand correctly, `+1` here means that the logs file will be considered from the start.

Suppose that the exporter crashed while squid did not, it will mean that the logs will be processed again from the start.

Now suppose that logs were rotated before the crash, the exporter will process the logs from "somewhere in the middle" rather than from the start, which will make a mess I believe.

Prometheus has some mechanisms to handle counter resets, so I think it would be better that we try to rely on those instead of trying to regenerate history ourselves.

cc @amisstea @avi-biton, how do you see that?